projects/dotcms-models/src/dot-contentlet/dot-contentlet.model.ts

```diff
@@ -0,0 +1,24 @@
+export interface DotCMSContentlet {
```

Comment:
DotCMSContentlet or DotContentlet?

Reply:
I followed the patter in the `dotcms-models` project to add `DotCMS` @fmontes please confirm.